README.md

```diff
@@ -68,6 +68,177 @@ Custom Secret Scanning Patterns repository.
 - Arc
   
 
+### [IBANs](./pii/generate_iban)
```

Comment:
This is strange. These shouldn't be all published. This might be an issue with the tool, let me take a look.

Reply:
I didn't notice that. Thank you 🙏 

I recall there was a flag 🏴 to set those to be hidden, and I may have not included that in my branch, or the `markdown` mode doesn't do that as you intended.

Reply:
I've added `display: false` to the country-level IBANs `patterns.yml`, which has fixed this